packages/framework/src/Model/Category/CategoryFacade.php

```diff
@@ -435,4 +438,33 @@ public function getListableProductCountsIndexedByCategoryId(
             $domainId
         );
     }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Category\Category $category
+     * @param array $originalNames
+     */
+    protected function createFriendlyUrlWhenRenamed(Category $category, array $originalNames): void
+    {
+        $this->friendlyUrlFacade->createFriendlyUrls(
```

Comment:
You should call this only when some names are available

Reply:
fixed